eval/eval.py

```diff
@@ -1748,21 +1748,21 @@ def process(fpath_and_category: Tuple[str, str]) -> Dict[str, Any]:
                                 # error with idx=dep_id
                                 dependencies.append((dep_id, error))
                             else:
-                                if QUIET:
+                                if not QUIET:
```

Comment:
This code was actually correct. Normally, eval.py prints its input and output via `bprint()`, and error messages are interspersed within the output. But if the QUIET option is set, the output is not printed; only the error messages are. In that case, we want to show the relevant file name before each error message. Thus the `if QUIET:` clauses.

Reply:
My bad! Didn't mean to push this, did this to help with repeated runs. Will fix!